masternodes/setup-testnet.rst

```diff
@@ -382,7 +381,7 @@ It should look like this when ready:
 
    Fully synchronized Dash Core wallet
 
-Click **Tools > Debug console** to open the console. Type the following
+Click **Window > Console** to open the console. Type the following
 command into the console to generate a new Dash address for the
 collateral::
 
```

Comment:
Link for Blockchain explorer:
<https://insight.dash.org/insight/>
should be replaced with:
<https://testnet-insight.dash.org/insight/>

Reply:
I would add also an information that CoinJoin should be stopped when sending 1000 Dash (it is not obvious and I faced a lot's of issues because of that)